back/src/dals/user/repository/user.mock.ts

```diff
@@ -95,7 +97,8 @@ export const getVotesFromRoom = (room: string) => {
   }));
 };
 
-export const freeRoom = (room: string) => {
+// TODO Here returns void because change the global variable
+export const freeRoom = async (room: string): Promise<void> => {
```

Comment:
You could return `Promsie<UserSession[]>` as we did in "real" repository and return the userCollectionSession